index.js

```diff
@@ -63,8 +63,13 @@
 //.
 //. The next step is to define a `def` function for the environment:
```

Comment:
This line has been separated from the code block meant to follow it by the heading below. Let's think of a way to provide API documentation for `$.create` without interrupting the flow of this tutorial.

Reply:
After thinking this over some more, I don't really think documenting the type signature of the `create` method has nearly the same value as documenting the type signature of the `def` module.  I completely agree with your comment and I think we should just revert it back.  I did add a bit more to line 64.  What are your thoughts?  
```suggestion
//. The next step is to define a `def` function for the environment using the
//. `create` method:
```

Reply:
This works for me, but for the fact that I consider `$.create` to be a function rather than a method. How about simply appending _using `$.create`_ to the sentence?

Reply:
So where you thinking something like this?

> The next step is to define a `def` function for the environment using `$.create`.

or where you thinking that we put the type signature in there?  So it would look something like this:

> The next step is to define a `def` function for the environment using `$.create`, which has the signature: `create :: { checkTypes :: Boolean, env :: Array Type } -> String -> StrMap (Array TypeClass) -> Array Type -> Function -> Function`.

I suppose we could combine the two options above into this:

> The next step is to define a `def` function for the environment using `$.create :: { checkTypes :: Boolean, env :: Array Type } -> String -> StrMap (Array TypeClass) -> Array Type -> Function -> Function`.

Reply:
I don't think providing the type signature of `$.create` would add value, given our intention to annotate `def` in the code block that follows.

Reply:
understood.  I will adjust and submit shortly.  Thank you.